src/assets/sass/protocol/_theme-config.scss

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// This file defines global theme settings for your project. You can
```

Comment:
Note for reviewer: I put this file in the root because I want to discourage consumers from editing any other files, especially since they're subject to change on future updates. Otherwise we could have defined these variables at the top of _lib, but I really don't like the idea of people messing with that file.

And defining them in protocol.scss wouldn't really work because it's really _lib that needs to know these values (they have to be defined _before_ _lib is imported) but maybe there's a better way.

Reply:
I think this is the best way to do it 👍

Reply:
Wait, can this file be ridden with a local copy? Or are they going to have to edit a file in the npm package?

Reply:
Crap you're right... it should be overridden locally, which means I need to rethink this... maybe we don't need a config file at all and we can just set the defaults at the top of _lib after all. I'll work on it.

Reply:
Well, these are identified as "default" in the config so the values should be over-rideable by a later declaration. In bedrock could we do it in the same place we do these definitions?

```
$font-path: '/media/fonts';
$image-path: '/media/protocol/img';
```

Reply:
Yeah, that's how it works, but I wrote out all these comments as if someone would modify their local copy... which you could totally do if you're just copying Protocol (like I was doing for all my testing), but when it's used as a package and this file lives in a node_modules folder nobody should be editing it and these comments are a bit misleading.

We do still have to declare _some_ default, but we can just put it right in _lib instead of in this separate file since we don't expect people to edit it, and I'll work on updating the comments and theme docs to explain how/where to change themes in a local file.